pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -861,6 +861,9 @@ func readPostgresStmt(
 			return unsupportedStmtLogger.log(fmt.Sprintf("%s", stmt), false /* isParseError */)
 		}
 		return wrapErrorWithUnsupportedHint(errors.Errorf("unsupported %T statement: %s", stmt, stmt))
+	case *tree.CreateType:
+		return errors.New("IMPORT PGDUMP does not support user defined types; please" +
```

Comment:
Although it says to do so in the error message, if a user doesn't remove references to the created types and only removes the `CREATE TYPE` statement, will we give an appropriate error during IMPORT? Might be worth adding a test case.

Reply:
Added a second commit to address this. Going to let CI run the importccl tests to see if I broke something unintentionally.